buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -13,9 +13,9 @@ object Versions {
   const val androidXTestExt = "1.1.1"
   const val androidXTest = "1.2.0"
   const val timber = "4.7.0"
-  const val dagger = "2.33"
+  const val dagger = "2.35.1"
   const val coreTesting = "2.0.0"
-  const val moshi = "1.11.0"
+  const val moshi = "1.12.0"
```

Comment:
Hmm, I would have preferred to do these in separate PRs instead of doing it in this one since errors can be caught sooner. This is fine for now, but lets try to make only one change per PR unless absolutely necessary.

Reply:
These are breaking changes, without these this PR would not compile. But yeah, I could have pulled out version bumps into a different PR and Kotlin change into another. Will remember for future such changes.

Reply:
Yeah, that's basically what I meant. If something breaks with these version changes, it is easier to make changes in a PR which only makes that change rather than one which also bundles other changes.